description.md

```diff
@@ -19,6 +19,35 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Drills
 
+Drills are educational activities designed to reinforce learning through repetitive practice.
+While quizzes are assessments aimed at testing understanding, drills involve repeated practice of a specific skill or concept.
+The primary goal of drills is to promote mastery and internalisation of skills or knowledge.
+By repeating tasks or exercises, students can reinforce their understanding and enhance their ability to apply learned concepts effectively.
+However, it's important to balance drills with other types of learning activities to maintain student engagement and prevent monotony.
+For example, drills can be combined with homework assignments, group discussions, and other activities to provide a more varied learning experience.
+The difference is that while drills are repetitive, simple, and focused on a single skill or concept, homework activities can be more complex and open-ended.
+
+Some key considerations when designing drills include:
```

Comment:
This list includes heterogeneous qualities. You could split these considerations into:
* What to keep in mind when designing drills - **Skill acquisition**, **Reinforcement**, **Buiding foundation**
* How are drills improving the flow of the learning process - **Confidence boost**, **Engagement**

Reply:
@Alex-deVis , I think this is something rather subjective. I would keep the list as it is. We can revisit it in the future.